e2e/utilities/utils.ts

```diff
@@ -156,25 +165,32 @@ export class Utils {
   }
 
   static async pressEscape() {
```

Comment:
can this be removed since it's been moved in component.ts ?

Reply:
still referenced by 36 files (test suits)